### PR TITLE
ensembl vep: fix requirements

### PR DESCRIPTION
**Pull request by bernt-matthias**
could not be resolved (some incompatibilities: 3.4 comes from bioconda 3.10 from conda-forge)

xref https://github.com/BioContainers/multi-package-containers/pull/2562

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/ensembl_vep/ensembl_vep.xml [hunk 1]
```diff
@@ -289,7 +289,7 @@
             <param name="cache_file" value="drosophila_melanogaster_vep_106_BDGP6.32" />
             <output name="output1" file="output_test1.vcf" ftype="vcf" lines_diff="2" />
         </test>
-        <test expect_num_outputs="2">
+        <!-- <test expect_num_outputs="2">
```
Comment (bgruening) [inline tools/ensembl_vep/ensembl_vep.xml:292]: taken the test out is what you want to do here?

Comment (bernt-matthias) [reply tools/ensembl_vep/ensembl_vep.xml:292]: Nah. Mistake. Just wanted to speed up local testing.

File: tools/ensembl_vep/ensembl_vep.xml [hunk 2]
```diff
@@ -354,15 +354,15 @@
             <param name="source" value="unrestricted" />
             <param name="input1" dbkey="dm6" value="input_test1.vcf" ftype="vcf" />
             <param name="cache_file" value="drosophila_melanogaster_vep_106_BDGP6.32" />
-            <param name="colo_var_checkboxes" value="--max_af,--pubmed" />
+            <param name="colo_var_checkboxes" value="\-\-max_af,\-\-pubmed" />
```
Comment (bernt-matthias) [inline tools/ensembl_vep/ensembl_vep.xml:357]:
```suggestion
            <param name="colo_var_checkboxes" value="--max_af,--pubmed" />
```

File: tools/ensembl_vep/ensembl_vep.xml [hunk 3]
```diff
@@ -341,7 +341,7 @@
             <param name="source" value="unrestricted" />
             <param name="input1" dbkey="dm6" value="input_test1.vcf" ftype="vcf" />
             <param name="cache_file" value="drosophila_melanogaster_vep_106_BDGP6.32" />
-            <param name="ident_checkboxes" value="--uniprot,--biotype" />
+            <param name="ident_checkboxes" value="\-\-uniprot,\-\-biotype" />
```
Comment (bernt-matthias) [inline tools/ensembl_vep/ensembl_vep.xml:344]:
```suggestion
            <param name="ident_checkboxes" value="--uniprot,--biotype" />
```

File: tools/ensembl_vep/ensembl_vep.xml [hunk 4]
```diff
@@ -328,7 +328,7 @@
             <param name="source" value="unrestricted" />
             <param name="input1" dbkey="dm6" value="input_test1.vcf" ftype="vcf" />
             <param name="cache_file" value="drosophila_melanogaster_vep_106_BDGP6.32" />
-            <param name="out_opt_checkboxes" value="--overlaps,--allele_number" />
+            <param name="out_opt_checkboxes" value="\-\-overlaps,\-\-allele_number" />
```
Comment (bernt-matthias) [inline tools/ensembl_vep/ensembl_vep.xml:331]:
```suggestion
            <param name="out_opt_checkboxes" value="--overlaps,--allele_number" />
```

File: tools/ensembl_vep/ensembl_vep.xml [hunk 5]
```diff
@@ -2,7 +2,7 @@
     <description>with VEP</description>
     <macros>
         <token name="@TOOL_VERSION@">109.3</token>
-        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bernt-matthias) [inline tools/ensembl_vep/ensembl_vep.xml:5]: Should we bump here, given then 0 did apparently never work?

**Review by wm75 — APPROVED** [review]

**Pull request merged**